var/spack/repos/spack_repo/builtin/packages/podio/package.py

```diff
@@ -102,6 +103,7 @@ class Podio(CMakePackage):
     depends_on("root@6.08.06: cxxstd=17", when="cxxstd=17")
     depends_on("root@6.14:", when="+datasource")
     depends_on("root@6.28.04: +root7", when="+rntuple")
+    depends_on("root@6.32: +root7", when="@1.3: +rntuple")
     depends_on("root@6.28:", when="@0.17:")
```

Comment:
```suggestion
    depends_on("root@6.28:", when="@0.17:")
    depends_on("root@6.32: +root7", when="@1.3: +rntuple")
```
To preserve ordering